### PR TITLE
feat(RELEASE-938): create-github-release writes a results file

### PR DESCRIPTION
**Pull request by johnbieren**
This commit changes the create-github-release task to write the created
github release url to a results file.

### CODE REVIEW

File: tasks/create-github-release/README.md [hunk 1]
```diff
@@ -13,6 +13,10 @@ a `release` dir.
 | repository | The github repository to release to | No | - |
 | release_version | The version string to use creating the release | No | - |
 | content_directory | The directory inside the workspace to find files for release | No | - |
+| resultsDirPath | Path to results directory in the data workspace | No | - |
+
+## Changes in 1.1.0
```
Comment (mmalina) [inline tasks/create-github-release/README.md]: The new parameter is mandatory, so it's a breaking change and should be 2.0.0.

Comment (johnbieren) [reply tasks/create-github-release/README.md]: Updated

**Review by mmalina — APPROVED** [review]

**Pull request merged**